index.js

```diff
@@ -23,6 +23,9 @@ const clear = moduleId => {
 
 	// Delete module from cache
 	delete require.cache[filePath];
+	Object.keys(require.cache).filter(key => new RegExp(moduleId).test(key)).forEach(key => {
```

Comment:
This is not a correct fix. You cannot just loosely match on a regex. That might even match non-node_modules directories.